### ISSUE TITLE
How can one edit the tags that are attached to annotations?

### ISSUE THREAD

**Issue by bobcat-hubcap** [elias-sundqvist/obsidian-annotator#346]
Over the years I find I have inadvertently duplicated some tags that I use to mark my annotations to pdfs. e.g Stupid things like a misspelling or using brackets when none are required. I would like to be able to view the list of tags and edit them. In a best case scenario, I would be able to apply edit to all instances as in changing the misspelled word in the tags would propagate it throughout in all other instances. 

So, how can I edit tags in the Annotator Plug-in? 

Many thanks.

**Comment by charlie-yi** [comment]
A good request. I have the same problem

**Comment by charlie-yi** [comment]
It is better that the app fills the label when  we choose one label, instead of adding this label directly to the annotation.

**Comment by bobcat-hubcap** [comment]
Yes, indeed. Would be great to have a plugin that could enter and manage keywords everywhere (including in these annotations). I quickly looked at the new(-ish) native pdf reader but didn't see the sort of functionality that this plugin provides. Was likewise timid about moving a couple thousand pdfs (with annotations) into the new reader. Took a look at hypothe.is but could see how keywords were handled there. Elias, do you know where the keywords get stored and how to access them? Many thanks. Obviously, your work is super-useful!

**Comment by ospknb** [comment]
I find myself in need of this as well. Any pointers on how to edit the tags in annotator it would be greatly appreciated.

**Comment by bobcat-hubcap** [comment]
I just opened a case on hypothes.is on this issue (although I don't use that service). I'll post anything I hear, should I hear anything. Elias? Thoughts?

**Comment by bobcat-hubcap** [comment]
Hypothes.is wrote back very quickly! Here's what they said:
"Thanks for writing. Tags are stored in two places, depending on which set of tags your viewing.

The actual tags stored on your annotations are stored in our database. To edit them you'd need to edit the annotations they're on. There's no easy way to bulk edit them, but there is a 3rd-party tool that will help you do so if that's what you want. It's tricky to use, however, so I'll pass along some details and warnings if that's what you want.

I suspect, however, that you want to edit the list of tags that appear in the drop down menu when you're adding tags to an annotation in the browser. That list of tags isn't loaded from our database, it's saved to your browser and consists of the tags you've used in the past in the browser, even if you've deleted them out of your annotations now.

If you want to just start over the easiest thing to do is clear your browser cache (being selective about what you delete, if you don't want to clear cached files from other sites). There's also ways of going into the browser cache and editing what's there."

So, Elias, can you confirm that this would also work for Obsidian which does not open in a browser but which still might have a similar cache architecture/functionality.

Many thanks.
